piksi_tools/console/gui_utils.py

```diff
@@ -130,3 +131,10 @@ def get_filter_group():
                         visible_when="{} in received_codes".format(code)))
             hgroup.content.append(vgroup)
         return hgroup
+
+
+class PiksiTabularAdapter(TabularAdapter):
```

Comment:
this looks great, piksi is a bit of a misnomer though, maybe `NoEditTabularAdapter`? Or something version that denotes what is being inherited

Reply:
There is some chance of further style etc. updates in the future. Anyway, renamed to `ReadOnlyTabularAdapter`, it is a descriptive name at the moment.